### ISSUE TITLE
Unhandled Exception for quickly created then deleted files

### ISSUE THREAD

**Issue by JLJTGR** [canton7/SyncTrayzor#79]
After upgrading to v1.0.17.0, I've been getting some crashes of the GUI when I task switch.  I attached SyncTrayzor to Visual Studio and captured the following exception:
```
System.IO.FileNotFoundException was unhandled
  _HResult=-2147024894
  _message=File C:\Users\<------->\.idea\workspace.xml___jb_bak___ not found
  HResult=-2147024894
  IsTransient=false
  Message=File C:\Users\<------->\.idea\workspace.xml___jb_bak___ not found
  Source=SyncTrayzor
  StackTrace:
       at SyncTrayzor.Utils.PathEx.GetLongPathName(String shortPath) in c:\Documents\projects\SyncTrayzor\src\SyncTrayzor\Utils\PathEx.cs:line 41
       at SyncTrayzor.Services.DirectoryWatcher.PathChanged(String path) in c:\Documents\projects\SyncTrayzor\src\SyncTrayzor\Services\DirectoryWatcher.cs:line 135
       at System.IO.FileSystemWatcher.CompletionStatusChanged(UInt32 errorCode, UInt32 numBytes, NativeOverlapped* overlappedPointer)
       at System.Threading._IOCompletionCallback.PerformIOCompletionCallback(UInt32 errorCode, UInt32 numBytes, NativeOverlapped* pOVERLAP)
  InnerException: 
```
Indeed, there is no file named that at the listed location.

According to ProcMon, when switching windows away from AndroidStudio, several workspace.xml "old" and "bak" files will be quickly created and deleted (1.6ms window).  

I suspect that the FileSystemWatcher is notifying you of these files being created, but when you get to the files they are gone and I guess you don't account for that.

**Comment by canton7** [comment]
Ah bollocks. It's actually any rename. I'll get that fixed....

**Comment by canton7** [comment]
Fixed in 1.0.18
